books/RayTracingInOneWeekend.html

```diff
@@ -647,29 +649,28 @@
 Ray-Sphere Intersection
 ------------------------
 <div class='together'>
-Recall that the equation for a sphere centered at the origin of radius $R$ is $x^2 + y^2 + z^2 =
-R^2$. Put another way, if a given point $(x,y,z)$ is on the sphere, then $x^2 + y^2 + z^2 = R^2$. If
-the given point $(x,y,z)$ is _inside_ the sphere, then $x^2 + y^2 + z^2 < R^2$, and if a given point
-$(x,y,z)$ is _outside_ the sphere, then $x^2 + y^2 + z^2 > R^2$.
+Recall that the equation for a sphere with a radius of $r$ and that is centered at the origin is
+$x^2 + y^2 + z^2 = r^2$. You can also think of this as saying that if a given point $(x,y,z)$ is on
+the sphere, then $x^2 + y^2 + z^2 = r^2$. If a given point $(x,y,z)$ is _inside_ the sphere, then
+$x^2 + y^2 + z^2 < r^2$, and if a given point $(x,y,z)$ is _outside_ the sphere, then
+$x^2 + y^2 + z^2 > r^2$.
 
-It gets uglier if the sphere center is at $(C_x, C_y, C_z)$:
+It gets uglier if the sphere center isn't at the origin, _i.e._ $(C_x, C_y, C_z)$:
```

Comment:
_Id est_ isn't correct here. Perhaps this is more clear?

It gets uglier if the sphere center is at some point $(C_x, C_y, C_z)$ that isn't the origin. In this case:

Reply:
Rewritten. I like it quite a bit more. Thanks for the eye